shame_script.py

```diff
@@ -151,8 +161,9 @@ async def fetch_and_send_tasks() -> None:
                 )
 
                 message_content.append(
-                    f"*Tasks for {discord_user.mention}*\n```\n{table}\n```"
+                    f"*Tasks for {discord_user.mention}* | Streak: {user_score.streak}\n```\n{table}\n```"
```

Comment:
breaking the formatting in the middle of the line doesn't look great, maybe should be in a new line?

Reply:
i think adding a newline for streak looks messier

heyitsme completed all tasks!
Streak: 1
rocket completed all tasks!
Streak: 1
snackwel completed all tasks!
Streak: 1

is doubling the size of this message, and there association between streak and user is a little less clear

maybe we can just keep the formatting?

Reply:
keeping the formatting seems fine